packages/wallet/hooks/useCreateAccounts.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { extractAddressFromPassphrase } from '@wallet/utilities/account';
+import { generatePassphrase } from '@views/utilities/passphrase';
+
+const useCreateAccounts = () => {
+  const [accounts, setAccounts] = React.useState([]);
```

Comment:
Since `useEffect` is destructured, `useState` should also be destructured

Reply:
okay